src/vcpkg/base/xmlserializer.cpp

```diff
@@ -90,6 +90,16 @@ namespace vcpkg
         }
         return *this;
     }
+    XmlSerializer& XmlSerializer::cdata(StringView sv)
```

Comment:
@ras0219-msft I think this CDATA thing was a workaround for not having real XmlSerializer available at the time; should this go through data() instead now that you have proper escapes implemented?

Reply:
The spec actually says "as a CDATA block": https://xunit.net/docs/format-xml-v2

(for the &lt;message> child of a &lt;failure> node)
> The failure message as a CDATA block.

Reply:
WOW!